tsdb/shard.go

```diff
@@ -1158,7 +1158,7 @@ func (s *Shard) CreateSnapshot() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return engine.CreateSnapshot()
+	return engine.CreateSnapshot(true)
```

Comment:
Is it okay to hard code this to `true`? Wondering if `Shard.CreateSnapshot()` also needs to take a flag.

Reply:
Shard.CreateSnapshot appears to only be called by tests in store_test.go and shard_test.go, so hardcoding to true maintains current behavior for those tests

Reply:
I think it's also called from outside of `influxdb`.

Reply:
Fixed in https://github.com/influxdata/influxdb/pull/19869/commits/6ec446f4222e60f07c698a973cd13a45553c0834